docs/deployment/deploy_with_existing_vineyard_cluster.md

```diff
@@ -36,12 +50,17 @@ python3 -m pip install vineyard
 ```
 
 By default, the Vineyard cluster consists of three Vineyard instances and three etcd instances. 
-However, since we only have one node in the Kubernetes cluster, we need to specify the number of Vineyard instances and etcd instances using the `vineyard_replicas` and `vineyard_etcd_replicas` parameters. 
+However, since we only have one node in the Kubernetes cluster, we need to specify the number of Vineyard instances and etcd instances using the `vineyard_replicas` and `vineyard_etcd_replicas` parameters. DON'T set the number of Vineyard instances and etcd instances to be greater than the number of nodes in the Kubernetes cluster.
+
```

Comment:
etcd replica could have a reasonable default, and don't let the user care those details in the getting started tutorial.

Reply:
Could we raise error when the number of vineyard instances greater than the number of nodes?

Reply:
> Could we raise error when the number of vineyard instances greater than the number of nodes?

It should be managed by the users rather than vineyardctl. Also, it won't block the creation of the vineyard cluster. if the number of vineyard instances is greater than the number of nodes, the other vineyard pods will be pending.